stac_fastapi/sqlalchemy/tests/conftest.py

```diff
@@ -152,3 +152,13 @@ def app_client(api_client, load_test_data, postgres_transactions):
 
     with TestClient(api_client.app) as test_app:
         yield test_app
+
+
+@pytest.fixture
+def app_client_2(api_client, load_test_data, postgres_transactions):
```

Comment:
Actually, small comment.  Instead of creating a new app client fixture, I think it would be better if we create the second collection in each test case (`test_create_item_duplicate`, `test_delete_item_duplicate`, and `test_update_item_duplicate`).

Reply:
@Jeff good call. I have done it, thanks!